src/components/atoms/CommentCard/index.js

```diff
@@ -1,30 +1,79 @@
-import React, { Fragment } from 'react';
-import { Link } from 'react-router-dom';
-import timePast from 'time_past';
-import _ from 'underscore';
-
-const CommentCard = (props) => {
-  const { comment } = props;
-  const name = `${comment.user.firstname} ${comment.user.lastname}`;
-  const userImage = 'https://res.cloudinary.com/dbsxxymfz/image/upload/v1536757459/dummy-profile.png';
-  return (
-    <Fragment>
-      <div id="comments-view">
-        <div className="comment-body">
-          {_.unescape(comment.content)}
-        </div>
-        <div className="comment-footer">
-          <img src={comment.user.image || userImage} alt="userImage" />
-          <p className="userName">
-            <Link to={`/users/${comment.user.id}`}>{name.trim() ? name : comment.user.username}</Link>
-          </p>
-          <p className="time-ago">
-            {timePast(comment.createdAt)}
-          </p>
+import React, { Component, Fragment } from "react";
+import { Link } from "react-router-dom";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import timePast from "time_past";
+import _ from "underscore";
+import {
+  likeComment,
+  getComments
+} from "Actions/commentActions";
+
+class CommentCard extends Component {
+  handleClick = async () => {
+    const { handleLike, articleId, commentId, fetchComments } = this.props;
+
+    await handleLike(articleId, commentId);
+
+    setTimeout(() => {
+      fetchComments(articleId);
+    }, 300);
```

Comment:
In a situation where we have a slow server/bad internet, fetchComments might execute first before handleLike finishes execution.

Look for a way to implement this synchronously in such a way that handleLike completes execution before fetchComments is called rather than relying on 300 timeout.